src/main/frontend/db/query_dsl.cljs

```diff
@@ -141,6 +141,31 @@
      l)
     @vars))
 
+;; TODO: Convert -> fns to rules
```

Comment:
It'll be nice when all these are rules so we can easily use these in advanced queries :)

Reply:
That's will be great.